platforms/core-runtime/base-services/src/test/groovy/org/gradle/internal/service/ScopedServiceRegistryTest.groovy

```diff
@@ -62,7 +80,7 @@ class ScopedServiceRegistryTest extends Specification {
 
         then:
         def exception = thrown(IllegalArgumentException)
-        exception.message.contains("Service '${BuildTreeScopedService.name}' was declared in scope 'BuildTree' but registered in scope 'Build'")
+        exception.message.contains("Service '${BuildTreeScopedService.name}' was declared in scopes [BuildTree] but registered in scope 'Build'")
```

Comment:
The original message was nicer for services that are declared in a single scope. Just saying...

Reply:
I thought so as well. I updated the code to have a catered message for the case of a single scope.